main/guides/getting-started/README.md

```diff
@@ -43,12 +48,29 @@ The Agoric SDK is supported on <a href="https://en.wikipedia.org/wiki/Linux">Lin
 - To launch a terminal on Linux, use the **Terminal** application.
 - To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
 
-## Install Node.js 16.19.1 or Higher
+## Install Go
 
-Download Node.js from [Node.js](https://nodejs.org/) and follow the instructions for your platform. We recommend installing the LTS version of node 16.
+Download Go from [go.dev/doc/install](https://go.dev/doc/install) and follow the instructions for your platform.
 
 ```shell
-node --version # 16.19.1 or higher
+go version
+```
+
+## Install jq
```

Comment:
`jq` should not be a required piece of getting started. what requires it?

Reply:
I agree that it _should_ not be required, but the proposed core eval docs (#799) use `make scenario2-setup` in `packages/cosmic-swingset` and that requires `jq`. As noted there, I'd like us to use `agoric start ...` instead of `make` but then we would have to more or less copy many of the commands out of the Makefile into the documentation. `scenario2-setup` does at least run in ci.

Reply:
Since the CoreEval docs now use the various agoric XXX commands rather than make scenario2-setup, I deleted all references to installing jq, since it's no longer required.